localization/english/wc_buildings_l_english.yml

```
@@ -22,11 +32,21 @@
  building_lordaeron_palace_01:1 "Lordaeron Palace"
  building_lordaeron_palace_01_desc:1 "The Palace of Lordaeron is the jewel of the Empire of Lordaeron and shows just how far humans have come. Its presence exudes an aura of prestige and with a hint of decadence."
  
+ building_type_nexus_01:0 "The Nexus"
+ building_type_nexus_01_desc:1 "The Nexus is an extensive series of caves and tunnels that riddles Coldarra."
+ building_nexus_01:1 "The Nexus"
+ building_nexus_desc:1 "The Nexus is an ancient ice fortress found in the center of Coldarra, an island in Borean Tundra. It is a column of magical energy surrounded by levitating earth-covered rings with ice caves underneath."
+ 
  building_type_ruined_palace_01:0 "Ruined Palace"
  building_type_ruined_palace_01_desc:1 "A city that spans beneath much of Lordaeron's ruin. A grave to kingdom."
  building_ruined_palace_01:1 "Ruined Palace"
  building_ruined_palace_01_desc:1 "The Ruins of Lordaeron is all that remains of the once glorious city of Lordaeron, the great capital city of the Kingdom of Lordaeron. Now it serves as a grim reminder of the power of the Scourge. The Ruined Palace still stands and its inhabitants are constantly haunted by the final tragic events that led to the death of King Terenas and the city's destruction."
  
+ building_type_shrine_of_fellowship_01:0 "Shrine Of Fellowship"
+ building_type_shrine_of_fellowship_01_desc:1 "A peaceful shrine up upon the peaks."
```

```suggestion
 building_type_shrine_of_fellowship_01_desc:1 "A peaceful shrine upon the peaks."
```

I think just leaving it at "upon the peaks" should be enough. 🤔